verifying/verifying.go

```diff
@@ -59,9 +59,35 @@ func VerifyVRFNonce(nonce *uint64, m *shared.VRFNonceMetadata, opts ...OptionFun
 	return nil
 }
 
+type ProofVerifier struct {
+	inner *postrs.Verifier
+}
```

Comment:
```suggestion
type ProofVerifier struct {
	*postrs.Verifier
}
```

With struct embedding `ProofVerifier` inherits the methods of `postrs.Verifier` (no need to define a `Close` method that just calls the inner `Close`)